packages/review-vir/src/ui/elements/main-page-elements/pull-requests-main-page/pull-request-presentation/vir-pull-request.element.ts

```diff
@@ -230,7 +230,7 @@ export const VirPullRequest = defineElement<{
                     <div class="rows grow">
                         <div class="columns center">
                             <span class="faint no-shrink">
-                                <a href=${inputs.pullRequest.branches.headBranch.repo.htmlUrl}>
+                                <a href=${inputs.pullRequest.branches.headBranch.repo.htmlUrl} target="_blank">
```

Comment:
Please also include `rel` for [security purposes](https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable):
```suggestion
                                <a href=${inputs.pullRequest.branches.headBranch.repo.htmlUrl} target="_blank" rel="noopener noreferrer">
```